contracts/chain-adapters/Rerouter_Adapter.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { AdapterInterface } from "./interfaces/AdapterInterface.sol";
+
+/**
+ * @notice Contract containing logic to send messages from L1 to a target (not necessarily a spoke pool) on L2.
```

Comment:
what do you mean by "not neccessarily a spoke pool?"

Reply:
Also, shouldn't this @notice explain why this contract is called a "re-router"? (because it sends messages from L1 to an L3 target by re-routing via an L2 contract.

Reply:
Addressed by other comments and this commit https://github.com/across-protocol/contracts/pull/607/commits/6d8a6339a81368fc47271c89581408d5d8b94f1b